dcos/dcos-ui
# DCOS-43897: fix table cell collapse

Removed min-width table-cell truncation hack because most browsers will obey the min-width: 0 style, causing the layout to break.

Tested in Firefox, Edge, Safari, and Chrome. Also tested to ensure text truncation still works.

## Testing
**Test in Firefox or Edge**
0. Check out `mp/fix/DCOS-43897-ff-table-cell` from dcos-ui-plugins-private
1. Create a new service
2. Go to the "Environment" tab
3. Click "+ Add Environment Variable" and enter a key and a value
4. Click "+ Add " and enter a key and a value
5. Click "+ Add " and enter a key and a value
6. Go to "Networking" tab
7. Click "+ Add Service Endpoint"
8. Go to "Health Checks" tab
9. Click "+ Add Health Check" and add enter dummy info
10. Click "Review & Run"

No columns in the table should be collapsed

## Trade-offs
This solution is a little bit of a table layout hack

## Dependencies
None

## Screenshots
Before:
<img width="933" alt="screen shot 2018-10-31 at 9 42 28 am" src="https://user-images.githubusercontent.com/2313998/47804387-e579e480-dcf1-11e8-9569-a47c127c1b81.png">


After:
<img width="921" alt="screen shot 2018-10-31 at 9 43 21 am" src="https://user-images.githubusercontent.com/2313998/47804393-e90d6b80-dcf1-11e8-98d4-802b05dc7ada.png">
